libenkf/src/enkf_main.c

```diff
@@ -1286,9 +1286,8 @@ bool enkf_main_UPDATE(enkf_main_type * enkf_main , const int_vector_type * step_
     int_vector_free( ens_active_list );
     return true;
   } else {
-    fprintf(stderr,"** ERROR ** There are %d active realisations left, which is less than the minimum specified (%d) - stopping assimilation.\n" ,
-            active_ens_size ,
-            analysis_config_get_min_realisations(analysis_config));
+    fprintf(stderr,"** ERROR ** There are %d active realisations left, which is less than the minimum specified - stopping assimilation.\n" ,
+            active_ens_size );
```

Comment:
did we lose the min_specified information in this context, or:  Why is it removed from the error message?